### ISSUE TITLE
[5.x] Unable to create stencils

### ISSUE THREAD

**Issue by ArnoMoqo** [verbb/formie#1794]
### Describe the bug

Whenever we want to create a new stencil we get the following error:

```php
Call to a member function getAttributes() on array
 
in /var/www/html/vendor/verbb/formie/src/models/Stencil.php
 
$projectConfig = Craft::$app->getProjectConfig();

if ($stencil->dateDeleted) {
    $configData = null;
} else {
    $configData = $stencil->getConfig();
}
````

### Steps to reproduce

1. Try to create a new stencil
2. See the error

### Form settings

clean install with default settings.

### Craft CMS version

5.0.0

### Plugin version

3.0.0-beta.5

### Multi-site?

No

### Additional context

No extra plugins, just a clean Craft 5 install with Formie.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.5"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.6](https://github.com/verbb/formie/releases/tag/3.0.0-beta.6)

**Comment by ArnoMoqo** [comment]
> Fixed in [3.0.0-beta.6](https://github.com/verbb/formie/releases/tag/3.0.0-beta.6)

There seems to be a new problem where the stencil does not safe correctly. Only the first tab gets saved, all other are being reset to the default start state.


https://github.com/verbb/formie/assets/117268595/1913d818-fbc2-486e-8bcd-1aa49dbddbf9

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.7"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.8](https://github.com/verbb/formie/releases/tag/3.0.0-beta.8)
